### DISCUSSION TITLE
Error log has no timestamp?

### DISCUSSION THREAD

**Discussion by givihuda** [dbeaver/dbeaver#13124] [General]
I got "Java" error while trying to "restore" .sql dump file to a database that I created. Normally this works no problem. However, often whenever there's an error, a pop-up box shows saying to check the error log with no further specifics; closing that box shows the underlying box now has a stacktrace that's unreadable to me (I can read Java and stacktrace but the text is not telling me informing me of any info that I know what to do with).

However, when I go to check the error log (Window > Show > Error Log), there's no timestamp on the lines, so I can't tell which ones apply to the issue I just had. Example:

![image](https://user-images.githubusercontent.com/74785256/124620256-8a9bd200-de36-11eb-9b5b-c5f0ca9860e9.png)

Now what? So far it's saying "it didn't work" or "you did something wrong" but I can't figure out why.

**Comment by LonwoLonwo** [comment]
Hello @givihuda 
You can see on the pictures on this wiki page https://github.com/dbeaver/dbeaver/wiki/Log-files that Error Log has timestamps. 
Also, you can use double-click on a specific error to get more information about it including a timestamp.

![2021-07-12 12_55_43-Event Details](https://user-images.githubusercontent.com/45152336/125269064-0606f300-e311-11eb-81ab-ebe1f85d8380.png)

Was my answer is useful for you?

**Comment by givihuda** [comment]
Oh, I see I had to scroll over to see the Date column. Thanks, I wasn't aware; another tab was obscuring the other columns and I didn't see the scrollbar at the bottom.
